### ISSUE TITLE
Saving query removes new lines

### ISSUE THREAD

**Issue by tomwys** [dimitri/pgcharts#11]
When you save query (not chart) on Firefox/Ubuntu all new lines are removed. This is problematic because well formatted query falls apart.

**Comment by dimitri** [comment]
Agreed, I think it's due to the code editor javascript: http://codemirror.net. Actually the way I use it, I guess. Still being very much a noob in javascript, I'd accept a hand here really... can you help?

**Comment by Kern3l** [comment]
It happens only on query page, not on charts page.

Seems connected with saving bug there - when you make multiple lines, on save it merges them without any spaces so it breaks the query. You need to add explicit space on the start/end of every line to save. If it's done in JS, then I suppose you're getting content by a single $('...').text() - for multiple matches it makes a merge of each text without any delimiters.
